test/lib/issue_tracker/test_pivotal.rb

```diff
@@ -141,6 +151,16 @@ def mock_find(story, story_id=nil)
         @project_mock.expect(:stories, @stories.expect(:find, story, [story_id]))
       end
 
+      def removed_branches
+        @removed_branches ||= {
+            'origin/branch1' => Branch::Base.from_hash({'ref' => 'branch1', 'remote' => 'origin', 'sha' => 'sha1', 'status' => 'success', 'created_at' => (Time.now - 3600), 'stories' => ['111']}),
+            'origin/branch2' => Branch::Base.from_hash({'ref' => 'branch2', 'remote' => 'origin', 'sha' => 'sha2', 'status' => 'removed', 'created_at' => (Time.now - 1800), 'stories' => ['222']}),
+            'origin/branch3' => Branch::Base.from_hash({'ref' => 'branch3', 'remote' => 'origin', 'sha' => 'sha3', 'status' => 'fail', 'created_at' => (Time.now - 1800), 'stories' => ['333']}),
+            'origin/branch4' => Branch::Base.from_hash({'ref' => 'branch4', 'remote' => 'origin', 'sha' => 'sha4', 'status' => nil, 'created_at' => (Time.now - 1800), 'stories' => ['444']})
```

Comment:
Done in 
https://github.com/FlashFunders/flash_flow/commit/cb172f855f77a65afbb4ddebefa0ca7f4e8f9323